### PR TITLE
[WIP] asu-nginx: add package

### PR DESCRIPTION
**Pull request by aparcar**
this packages contains a nginx configuration to simplify the
installation of the attended sysupgrade server (asu).

Signed-off-by: Paul Spooren <mail@aparcar.org>

### CODE REVIEW

File: devel/asu-nginx/Makefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    DEPENDS:=+asu +nginx
+endef
+
+define Package/$(PKG_NAME)/description
+   Meta package to automate the configuration of asu behind a nginx server
+endef
+
+define Package/$(PKG_NAME)/install
+	$(INSTALL_DIR) $(1)/etc/nginx/sites-enabled/
+	$(INSTALL_BIN) ./files/asu-server.conf $(1)/etc/nginx/sites-enabled/
```
Comment (dangowrt) [inline devel/asu-nginx/Makefile]: Shouldn't this rather go to sites-available and then have symlinks in sites-enabled which can be created eg. by a uci-defaults script?

**Comment by BKPepe** [comment]
ping @aparcar

**Comment by BKPepe** [comment]
@aparcar how it goes with this PR?

**Comment by aparcar** [comment]
Thanks for the reminder. I'll close this for now until `asu` is in a stable state

**Pull request closed without merging**